packages/components/timepicker/package.json

```diff
@@ -23,23 +23,20 @@
     "build": "parcel build index.ts"
   },
   "peerDependencies": {
-    "react": ">=16.8"
+    "react": ">=16.8",
+    "react-dom": ">=16.8"
   },
   "devDependencies": {
-    "@contentful/forma-36-react-components": "^3.73.2",
-    "@contentful/forma-36-tokens": "^0.11.2",
     "@testing-library/jest-dom": "5.15.0",
     "@testing-library/react": "12.1.2",
-    "@types/is-hotkey": "^0.1.1",
-    "@types/jest": "25.2.3",
-    "@types/lodash.orderby": "^4.6.6"
+    "@types/jest": "25.2.3"
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
-    "@contentful/forma-36-react-components": "^3.100.7",
-    "date-fns": "^2.9.0",
+    "@contentful/f36-components": "^4.24.1",
```

Comment:
it's like "back to the future" component from V3 uses components from V4 😄

Reply:
Doesn't that mean when the user interface uses a different version it will include both?
Why don't we move all of this to a new package on `main` v4 and mark it as `v1.0.0` or even `v4.0.0-alpha`